crash_similarity.py

```diff
@@ -123,6 +127,32 @@ def train_model(corpus):
     return model
 
 
+def wmdistance(model, doc_words, words_to_test_clean, all_distances):
```

Comment:
Can you add a comment on top here that says this code was modified from https://github.com/RaRe-Technologies/gensim/blob/4f0e2ae/gensim/models/keyedvectors.py#L339?
This way if gensim changes something in the future, we can refer to their code.